source/api/presentation/3.0/change-log.md

```diff
@@ -1,125 +1,263 @@
+
+#### 1.3.6. Canvases always include AnnotationPages, not Annotations directly
+
+For consistency, and to allow all content to be external to the Manifest, all Annotations are now only included via `AnnotationPage` resources. This is a change from previous versions, where image Annotations were included directly to Canvases using `images`. See issue [#1068](https://github.com/IIIF/api/issues/1068).
+
+#### 1.3.7. Change requirements for `navDate` value
+
+Previous, the value of `navDate` was required to be in the UTC timezone. However this meant that the navigation date text generated was sometimes not the same date as the resource due to when midnight occurs in the local timezone versus in UTC. The solution adopted was to allow any timezone to be given in the value. See issue [#1296](https://github.com/IIIF/api/issues/1296).
+
+### 1.4. Classes Changes
+
+#### 1.4.1. Remove Sequence in favor of Ranges, `items` and `behavior` value `sequence`
+
+There has been a long outstanding question of how the order of Canvases in the `items` of Ranges and the order in the current Sequence interact. With further implementations from outside of the initial library-oriented domain, the need for an explicit order for views is also greatly decreased. The initial use case of multiple orderings for the content remains valid, but an edge case that can be supported using other patterns rather than requiring every Manifest to have an explicitly named Sequence. The replacement is a new `behavior` value or `sequence`. The model is simplified and clarified by this removal. See issues [#1471](https://github.com/IIIF/api/issues/1471), [#1489](https://github.com/IIIF/api/issues/1489) and further issues referenced from those.
+
+#### 1.4.2. Ranges express full extent
+
+In previous versions, it was unclear whether Ranges were intended to capture the extent of the content or simply the hierarchy. This is particularly important with time-based media, as the intent is not to skip between the navigation entry points when rendering the Range, but instead to play continuously. However a Range that captures the discrete sections of newspaper pages that make up an article does not include all of the intervening content.  This difference was reconciled by requiring Ranges to include all and only Canvases or segments of Canvases that are strictly part of the Range. At the same time, the structure was made easier to process by embedding child ranges within the parent, rather than referencing within a flat list.
+See issue [#1070](https://github.com/IIIF/api/issues/1070) and the referenced documentation.
+
+#### 1.4.3. Remove Layer, AnnotationList in favor of AnnotationCollection, AnnotationPage
+
+With the adoption of the Web Annotation Data Model, we remove the IIIF specific Layer in favor of the standard AnnotationCollection and AnnotationList in favor of the equivalent AnnotationPage. See issue [#496](https://github.com/IIIF/api/issues/496).
```

Comment:
A comma helps the flow here a bit I think:

"...Layer in favor of the standard AnnotationCollection, and AnnotationList in favor of..."

Reply:
:+1: